packit_service/service/api/copr_builds.py

```diff
@@ -127,10 +123,7 @@ def get(self, id):
                 # Get status per chroot as well
                 build_dict["status_per_chroot"][sbid_build.target] = sbid_build.status
 
-            build = make_response(dumps(build_dict))
-            build.headers["Content-Type"] = "application/json"
-            build.headers["Access-Control-Allow-Origin"] = "*"
-            return build if build else ("", HTTPStatus.NO_CONTENT)
+            return response_maker(build_dict)
 
         else:
-            return "", HTTPStatus.NO_CONTENT
+            return response_maker([])
```

Comment:
Hm.... wouldn't "not found" be a better response here? Why was "no content" used in the previous implementation?

Reply:
> Why was "no content" used in the previous implementation?
Yes, previous implementation was wrong, no content should not be used here.

Also, from what Ive read its a good practice to have the same json structure at the same endpoint.
So an empty list here makes more sense compared to an error message.  

Similar case here https://softwareengineering.stackexchange.com/a/358245